src/Common/src/CoreLib/System/IO/FileStream.Win32.cs

```diff
@@ -19,8 +19,8 @@ private unsafe SafeFileHandle CreateFileOpenHandle(FileMode mode, FileShare shar
             Interop.Kernel32.SECURITY_ATTRIBUTES secAttrs = GetSecAttrs(share);
 
             int fAccess =
-                ((_access & FileAccess.Read) == FileAccess.Read ? GENERIC_READ : 0) |
-                ((_access & FileAccess.Write) == FileAccess.Write ? GENERIC_WRITE : 0);
+                ((_access & FileAccess.Read) == FileAccess.Read ? Interop.Kernel32.GenericOperations.GENERIC_READ : 0) |
+                ((_access & FileAccess.Write) == FileAccess.Write ? Interop.Kernel32.GenericOperations.GENERIC_WRITE : 0);
```

Comment:
This file is shared with CoreLib. If you want to clean it up in this file, you should also move GenericOperations under src/Common/src/CoreLib/ and include it System.Private.CoreLib.Shared.projitems

Reply:
Ah okay thanks for letting me know. I will revert this change and take care of this separately to unblock this PR.

Reply:
I created this bug to track this: https://github.com/dotnet/corefx/issues/42199